### PR TITLE
[Feature][Olana] Winners in homepage

### PR DESCRIPTION
**Pull request by jrolana**
image

### CODE REVIEW

File: home/index.php [hunk 1]
```diff
@@ -107,8 +107,20 @@
                     echo "<li>" .
                         "<h3>" . $hackathon["theme"] . "</h3>" .
                         "<p>" . $hackathon["description"] . "</p>" .
-                        "<p>" . $hackathon["dateStart"] . " to " . $hackathon["dateEnd"] . "</p>" .
-                        "</li>";
+                        "<p>" . $hackathon["dateStart"] . " to " . $hackathon["dateEnd"] . "</p>";
+
+                    $winner_sql = "SELECT title, username FROM projects NATURAL JOIN users
```
Comment (ezerssss) [inline home/index.php]: use inner join tapos on projecs.winning = users.projectid

Comment (jrolana) [reply home/index.php]: whats projects.winning?

Comment (jrolana) [reply home/index.php]: wala ring projectID column sa users

Comment (ezerssss) [reply home/index.php]: AY JOKE

dapat triple join pala to


(hackathons inner join projects on hackathons.winningProjectID == projects.projectId) inner join users using userID

**Review by ezerssss — APPROVED** [review]

**Pull request merged**